game/services.py

```
@@ -68,47 +147,101 @@ def cleanup_stale_games():
+
+            try:
+                session = Session.objects.get(
+                    session_key=active_game.session_key
+                )
+                session_data = session.get_decoded()
+            except Session.DoesNotExist:
+                active_game.delete()
+                deleted_count += 1
+                continue
+            except Exception:
+                continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)